docker-assets/README.md

```diff
@@ -11,6 +11,13 @@ It needs to be initiated from the root directory of the manageiq git repository
 docker build -t manageiq .
 ```
 
+To build versions other than the tip of master:
+
+```
+docker build -t manageiq:darga --build-arg REF=darga .                 # From tip of darga branch
+docker build -t manageiq:darga-1-beta1 --build-arg REF=darga-1-beta1 . # From darga-1-beta1 tag
+```
+
 The image has been tested and validated under docker-1.10 (Fedora23) and 1.8.2 (Centos7)
```

Comment:
We need to remove the 1.8.2 (Centos7) , build ARGs are not implemented in that release.